Documentation/APIs.md

```diff
@@ -170,6 +171,26 @@ Easily add color layer on top of the UI element especially `AnimatableImageView`
 | ------------- |:-------------:| ----- |
 | hideNavigationBar | Bool | whether to hide navigation bar. The default value is `false`. |
 
+#### `RefreshControlDesignable`
+Display a `UIRefreshControl` in `AnimatableTableViewController`
```

Comment:
I wouldn't add this line, I think the protocol name is clear enough (moreover, we may conform to this protocol in other classes).